stock_env.py

```diff
@@ -39,8 +40,8 @@ def __init__(self,
         self.borrow_interest = borrow_interest_annual / days_per_year  # daily penalty on borrowed stocks
         self.transaction_penalty = transaction_penalty
 
-        data = get_data()  # pricing data
-        self.pricing_data = data[1] if is_testing else data[0]
```

Comment:
why is the `is_testing` arg being taken out? I thought this was some kind of train/test split